fluentd/configs.d/openshift/filter-syslog-record-transform.conf

```diff
@@ -32,6 +32,6 @@
     # if field is optional you have to use record[name] or (name rescue nil) or something like that
     ident ${record["SYSLOG_IDENTIFIER"] || record["_COMM"]}
     message ${record["MESSAGE"]}
-    version 1.3.0
+    version 1.3.1
```

Comment:
This originally also had the changes to the default time being shipped to ES, that was why the version was bumped.  Since that is part of master now i'll drop the version back down (I was anticipating that change going out first here)